config/clusters/iam.tf

```
@@ -218,6 +218,44 @@ data "aws_iam_policy_document" "rules_s3_access" {
   }
 }
 
+# Test-infra repository
+
+module "test-infra_s3_role" {
+  source    = "terraform-aws-modules/iam/aws//modules/iam-github-oidc-role"
+  version = "5.10.0"
```

```suggestion
  version = "5.10.0"
  name = "github_actions-test-infra-s3"
```
Just a nit but if you add this the role name will be predictable!